### DISCUSSION TITLE
How do I make the ComboBox individual options highlight on mouse hover?

### DISCUSSION THREAD

**Discussion by b-fuze** [slint-ui/slint#2893] [Q&A] [answered]
So I'm using Slint via the Qt backend with Rust, but when I hover my Combobox the individual options don't respond to mouse hovering:

https://github.com/slint-ui/slint/assets/10504957/4e65470b-949a-49f2-9a5b-38487c460ba4

I have observed this on both macOS Ventura and Ubuntu Linux 22.04.

How do I get it to behave similar to other Qt applications and highlight each individual option on mouse hover, e.g. VirtualBox's dropdown for instance?

https://github.com/slint-ui/slint/assets/10504957/1a64cbca-8749-415f-8406-ab4dac05e90e

**Comment by ogoffart** [comment, marked as answer]
We were not drawing the ComboBox popup correctly with all styles.
I've just fixed that in https://github.com/slint-ui/slint/pull/2897

**Reply by b-fuze** [reply]
Awesome, thanks!
